src/lib/constants.js

```diff
@@ -0,0 +1,9 @@
+//Import Assets
+import Image from "../assets/images/sushiMochi.jpg";
```

Comment:
Preferencia por usar `absolute imports`
`import Image from ".assets/images/sushiMochi.jpg";`
```suggestion
import Image from "assets/images/sushiMochi.jpg";
```

Reply:
Done!